### DISCUSSION TITLE
Inline math in visual editor is not displayed

### DISCUSSION THREAD

**Discussion by mariannasebo** [quarto-dev/quarto-cli#3348] [Q&A]
Hi! I am writing a quarto file with docx as output. I would like to insert inline math in the text. When I am using the visual editor it doesn't get displayed. Looking at the source, it is there, also in the output file it is rendered correctly. Does someone know how I could make it displayed also in the visual editor? Thanks!

**Comment by mcanouil** [comment]
This seems to be something that should be reported in <https://github.com/rstudio/rstudio>.
